ui/page/components/utils.go

```diff
@@ -12,3 +17,22 @@ func ContextDone(ctx context.Context) bool {
 		return false
 	}
 }
+
+// RetryFunc implement retry policy for processes that needs to be executed
+// in the background after initial failure.
+func RetryFunc(retryAttempts int, sleepDur time.Duration, errFunc func() error) (int, error) {
+	var err error
+	for i := 0; i < retryAttempts; i++ {
+		if i > 0 {
+			sleepDur *= 2
```

Comment:
```suggestion
			sleepDur *= (2-i)
```
So when i == 1, sleepDur is sleepDur *= 1 and only begins to increase when i > 1.

Reply:
invalid operation: sleepDur *= 2 - i (mismatched types time.Duration and int)